M2/Macaulay2/packages/A1BrouwerDegrees.m2

```diff
@@ -1,104 +1,101 @@
 --A1BrouwerDegrees.m2
-newPackage(
+newPackage (
     "A1BrouwerDegrees",
-    Version=>"1.0",
-    Date=>"June 5, 2023",
-    Authors=>{
-        {Name=>"Nikita Borisov",
-	 Email=>"nborisov@sas.upenn.edu",
-	 HomePage=>"https://www.math.upenn.edu/people/nikita-borisov"},
-        {Name=>"Thomas Brazelton",
-	 Email=>"brazelton@math.harvard.edu",
-	 HomePage=>"https://tbrazel.github.io/"},
-        {Name=>"Frenly Espino",
-	 Email=>"frenly@sas.upenn.edu",
-	 HomePage=>"https://www.math.upenn.edu/people/frenly-espino"},
-         {Name=>"Tom Hagedorn",
-	 Email=>"hagedorn@tcnj.edu",
-	 HomePage=>"https://hagedorn.pages.tcnj.edu/"},
-        {Name=>"Zhaobo Han",
-	 Email=>"zbtomhan@sas.upenn.edu",
-	 HomePage=>"https://www.linkedin.com/in/zhaobo-han-77b1301a2/"},
-     	{Name=>"Jordy Lopez Garcia",
-	 Email=>"jordy.lopez@tamu.edu",
-	 HomePage=>"https://jordylopez27.github.io/"},
-        {Name=>"Joel Louwsma",
-	 Email=>"jlouwsma@niagara.edu",
-	 HomePage=>"https://www.joellouwsma.com/"},
-        {Name=>"Andrew Tawfeek",
-	 Email=>"atawfeek@uw.edu",
-	 HomePage=>"https://www.atawfeek.com/"},
-        {Name=>"Wern Juin Gabriel Ong",
-	 Email=>"gong@bowdoin.edu",
-	 HomePage=>"https://wgabrielong.github.io/"}
+    Version => "1.1",
+    Date => "July 23, 2024",
+    Authors => {
+        {Name => "Nikita Borisov",
+	 Email => "nborisov@sas.upenn.edu",
+	 HomePage => "https://www.math.upenn.edu/people/nikita-borisov"},
+        {Name => "Thomas Brazelton",
+	 Email => "brazelton@math.harvard.edu",
+	 HomePage => "https://tbrazel.github.io/"},
+        {Name => "Frenly Espino",
+	 Email => "frenly@sas.upenn.edu",
+	 HomePage => "https://www.math.upenn.edu/people/frenly-espino"},
+         {Name => "Tom Hagedorn",
+	 Email => "hagedorn@tcnj.edu",
+	 HomePage => "https://hagedorn.pages.tcnj.edu/"},
+        {Name => "Zhaobo Han",
+	 Email => "zbtomhan@sas.upenn.edu",
+	 HomePage => "https://www.linkedin.com/in/zhaobo-han-77b1301a2/"},
+     	{Name => "Jordy Lopez Garcia",
+	 Email => "jordy.lopez@tamu.edu",
+	 HomePage => "https://jordylopez27.github.io/"},
+        {Name => "Joel Louwsma",
+	 Email => "jlouwsma@niagara.edu",
+	 HomePage => "https://www.joellouwsma.com/"},
+        {Name => "Wern Juin Gabriel Ong",
+	 Email => "gong@bowdoin.edu",
+	 HomePage => "https://wgabrielong.github.io/"},
+        {Name => "Andrew Tawfeek",
+	 Email => "atawfeek@uw.edu",
+	 HomePage => "https://www.atawfeek.com/"}
 	},
-    Headline=>"for working with A1-Brouwer degree computations",
-    PackageImports=>{
-	"Parametrization",
-	"RealRoots",
-	"RationalPoints2"
-	},
-    PackageExports=>{},
-    AuxiliaryFiles=>true,
-    Keywords => {"Commutative Algebra"}
```

Comment:
I added this keyword in #3216, but it was just a guess and perhaps there's something more appropriate.